### PR TITLE
feat: Use release-automation

### LINKED ISSUE #123: Use release automation
Use release automation to do releases https://github.com/nearform/optic-release-automation-action

### PR DESCRIPTION
**Pull request by ja-bravo**
Not ready to be merged as it depends on https://github.com/nearform/optic-release-automation-action/pull/30

Closes #123

### CODE REVIEW

**Review by simoneb — APPROVED** [review]
perfect. as mentioned, let's wait to merge until the release automation action has finally decided a way to allow users to carry out build steps

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          node-version: 16
+
+      - uses: nearform/optic-release-automation-action@v2.4.0
+        with:
+          github-token: ${{ secrets.github_token }}
+          semver: ${{ github.event.inputs.semver }}
+          sync-semver-tags: true
+          build-command: |
+            npm ci
+            npm run build
```
Comment (Eomm) [inline .github/workflows/release.yml:29]: I like adding an `echo` to make it explicit what we are doing

The installation is enough 👍🏽 

```suggestion
          build-command: |
            echo 'installing husky hooks..'
            npm ci
```

Here is an example:
https://github.com/nearform/optic-release-automation-action/pull/39

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Optionally configure your preferred runtime
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - uses: nearform/optic-release-automation-action@v2.4.0
```
Comment (Eomm) [inline .github/workflows/release.yml:22]:
```suggestion
      - uses: nearform/optic-release-automation-action@v2
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        required: true
+        default: 'patch'
+  pull_request:
+    types: [closed]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Optionally configure your preferred runtime
```
Comment (Eomm) [inline .github/workflows/release.yml:17]:
```suggestion
      - name: Node.js runtime on build
```

**Review by Eomm — APPROVED** [review]

**Pull request merged**